### PR TITLE
fix: consider empty but required fields

### PR DESCRIPTION
**Pull request by fkm**
Not sure if that fixes everything. But it would at least be a start :smile:

### CODE REVIEW

File: addon/lib/document.js [hunk 1]
```diff
@@ -185,7 +185,14 @@ export default EmberObject.extend({
       }
 
       const visibleFields = this.fields.filter(f => !f.question.hidden);
-      if (visibleFields.every(f => f.isValid)) {
+      const requiredFields = visibleFields.filter(
+        f => f.question.isRequired !== "false"
```
Comment (anehx) [inline addon/lib/document.js]: Isn't `question.isRequired` a JEXL expression?

Comment (fkm) [reply addon/lib/document.js]: From it being a string; that's possible. Let me check if I can find the right property.

Comment (czosel) [reply addon/lib/document.js]: As discussed, should be `optional`

Comment (fkm) [reply addon/lib/document.js]: Done

**Pull request merged**